modules/visualization/dcv-image/src/update_parameters.py

```diff
@@ -0,0 +1,93 @@
+import json
+import logging
+import os
+import subprocess
+import sys
+import typing
+
+import boto3
+import kubernetes
+
+NAMESPACE_FILE_PATH = "/var/run/secrets/kubernetes.io/serviceaccount/namespace"
+ADDF_CONFIG_MAP_ENV_VAR_NAME = "ADDF_CONFIG_MAP_NAME"
+ADDF_APPLICATION_NAMES_ENV_VAR_NAME = "ADDF_APPLICATION_NAMES"
+ADDF_DCV_SESSION_INFO_DISPLAY_KEY = "x11-display"
+ADDF_DCV_DISPLAY_KEY_SUFFIX = "display"
+ADDF_CONFIG_MAP_PATCH_KEY = "data"
+ADDF_DCV_SESSION_NAME = "default-session"
+ADDF_SSM_PARAMETER_STORE_DISPLAY_NAME = "dcv-display"
+
+
+def get_display_number() -> str:
```

Comment:
Can you pls add docstring to the functions in this file?

Reply:
Added.